### ISSUE TITLE
what's the acc meaning?

### ISSUE THREAD

**Issue by leoli2012** [tflearn/tflearn#1107]
When I run the tfLearn example, i get logs printing acc ,but I am not sure what acc identifies exactly.
Does "acc" mean the accuracy of all of training data or batch size data?
And what is val acc? compute all data in validation data or validation_batch_size?

Thanks in advance
